docker-compose.yml

```diff
@@ -13,3 +13,36 @@ services:
       - /app/.next
     ports:
       - '3000:3000'
+    environment:
+      - MONGO_URL=mongodb://mongo:27017
+      - MONGODB_DB=dev
+    stdin_open: true
+    depends_on:
+      - mongo
+  mongo:
+    container_name: mongo
+    image: mongo:5.0.3
```

Comment:
This is too new and we risk incompatibility with DocumentDB

Need 3.6 or 4.0 https://aws.amazon.com/documentdb/faqs/ 
https://docs.aws.amazon.com/documentdb/latest/developerguide/mongo-apis.html

Reply:
we'll be using 4